crates/rnote-compose/src/lib.rs

```diff
@@ -31,6 +31,8 @@ pub mod style;
 pub mod transform;
 /// other misc utilities
 pub mod utils;
+/// vertical tool options
+pub mod verticaltoolconfig;
```

Comment:
Why is this in rnote-compose? I think it should be in rnote-engine under `pensconfig` (and renamed to verticalspacetoolconfig`